config/ranks/bsg13.txt

```diff
@@ -0,0 +1,52 @@
+Captain=CMDR
+Executive Officer=COL
+Head of Security=LTCOL
+Chief Engineer=LTCOL
+Research Director=LTCOL
+Chief Medical Officer=LTCOL
+Master At Arms=MAJ
+Warden=MAJ
+
+Bridge Staff=LT
+
+Assistant=ENS
+
+Quartermaster=MAJ
+Cargo Technician=LT2
+Shaft Miner=SPC
+
+Bartender=CRW
+Cook=CRW
+Botanist=CRW
+Janitor=CRW
+
+Clown=CRW
+Mime=CRW
+Curator=CRW
+Lawyer=CRW
+
+Chaplain=CRW
+
+Station Engineer=LT
+Atmospheric Technician=LT
+
+Medical Doctor=LT
+Paramedic=LT2
+Chemist=LT2
+Geneticist=LT2
+Virologist=LT2
+
+Scientist=LT
+Roboticist=LT
+
+Detective=LT1
+Security Officer=LT
+Brig Physician=LT2
+
+Flight Leader=CPT
```

Comment:
Might be a bit confusing having a captain on a ship that isn't the captain. Suggest bump this to Major if you don't just refactor.